task/02-numbers-tasks.js

```diff
@@ -89,7 +90,15 @@ function getDistanceBetweenPoints(x1, y1, x2, y2) {
  *   5*x = 0         => 0
  */
 function getLinearEquationRoot(a, b) {
-    throw new Error('Not implemented');
+    let c = (b)/a
+    let z= (-b)/a
+    if(c!==(0)){
+        return z
+    }
+    if (c===(0))
+    {
+        return c
+    }
```

Comment:
![Selection_002](https://user-images.githubusercontent.com/26328278/92889350-d05a5a80-f41e-11ea-933b-13b343a482f9.png)
curly bracket at the same lin as if, space before curly bracket